task_runner

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
This should be `#!/usr/bin/env bash` for portability (see this [SO answer](http://stackoverflow.com/a/10383546/3776794) for details).